rest-api/tools/download_data_files.py

```diff
@@ -0,0 +1,24 @@
+import os
+""" Downloads csv files from Google buckets and places in data/ dir (gitignored).
+"""
+
+# @todo: change this to the new proper bucket once it's made
+SOURCE_BUCKET = 'all-of-us-rdr-sequestered-config-test/'
```

Comment:
Do we need to keep `export_organizations.sh` before we have the proper bucket name?

Reply:
good point. I put it back.